src/app.js

```diff
@@ -1,21 +1,28 @@
+/* eslint-disable no-console */
+// node ./src/app.js ./testMove/test.txt ./testMove/init/test.txt
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const fs = require('fs');
 
-module.exports = sum;
+const [currentPath, newPath] = process.argv.slice(2);
```

Comment:
In this case, the names are chosen a little incorrectly, the first argument will not contain the current path, but the name of the file, and the second argument will not always be a new path, but may be a new name, so it is better to choose the following names:
```
const [fileName, param] = process.argv.slice(2);
```